docs/guide/cli-service.md

```diff
@@ -92,6 +92,32 @@ There are a few useful flags:
 
 - `--report` and `--report-json` will generate reports based on your build stats that can help you analyze the size of the modules included in your bundle.
 
+## vue-cli-service lint
```

Comment:
**concern**: I think this section of the CLI docs describes only scripts available with the basic setup, i.e. without any CLI plugins included. However, `lint` script is added only with `@vue/cli-plugin-eslint`. My suggestion would be to move this part to respective docs (i.e. `@vue/cli-plugin-eslint` documentation).

Also, PRs to docs should be created using `master` or `next` branch as a base (see [Contributing guide](https://github.com/vuejs/vue-cli/blob/dev/.github/CONTRIBUTING.md)).

In `master` ESLint plugin docs are located [here](https://github.com/vuejs/vue-cli/blob/master/packages/%40vue/cli-plugin-eslint/README.md)

In `next` we changed them to be the part of the [main documentation](https://github.com/vuejs/vue-cli/blob/next/docs/core-plugins/eslint.md) rather than standalone READMEs.

Reply:
Whoa that's weird. I just used the "Edit" link on the page to do this. That's interesting that it made the destination vuejs:docs rather than master (which is what I assumed).

Shall I just make the PR for `next` then?

Reply:
Oh, that's a great catch! I will fix the Vuepress links to the repo! 🙇‍♀ 

Yes, I think a PR for `next` should work 😉